assets/img/icons/exit-icon-primary-darker.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Correct. Alex rebuilt that image as an SVG recently so we could translate it to different colors. The hover state color was new and is getting added here.